samples/msal-node-samples/managed-identity/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
```

Comment:
That's a lot of dead code. Is it needed?

Reply:
Nope, it was auto-generated. I can delete it.